### ISSUE TITLE
Fix reading an existing audio

### ISSUE THREAD

**Issue by BeltonAlberto** [Language-Tools/anki-hyper-tts#153]
When Adding new audio e.g. to the front/back it also reads every letter of the existing audio tag. It generates new audio and reads the sentence including what's within the previous audio tag.
It should generate an audio ignoring what is within [xxxxxxxxxx]
![An Issue](https://github.com/Language-Tools/anki-hyper-tts/assets/89537405/5e5c2b0a-f70d-4862-ac1d-fb250f5f4196)

**Comment by luc-languagetools** [comment]
Can you show me a screenshot of the "text processing" tab when you configure your preset ? This shouldn't be happening, the existing sound tag should be ignored.

**Comment by BeltonAlberto** [comment]
![Belton Menete's Text processing Anki preset settings](https://github.com/Language-Tools/anki-hyper-tts/assets/89537405/055fa709-6e16-4d1c-80eb-c75c0ffa58fb)

**Comment by BeltonAlberto** [comment]
Thanks for getting back to me. I took another look at the "text processing" tab, and it turns out the 'remove texts in brackets (), [], {}, <>' option wasn't checked by default. My bad for missing that!

I've now checked the option, and it fixed the issue with the existing sound tag being included in the new audio generation. Everything seems to be working as expected.

Thanks for your help, and sorry for any confusion. Let me know if there's anything else I should do.
